scrapers/TheFlourish.yml

```
@@ -0,0 +1,41 @@
+name: The Flourish
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - theflourishxxx.com/trailers/
+      - theflourishamateurs.com/trailers/
+      - theflourishfetish.com/trailers/
+      - theflourishpov.com/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[contains(@class, 'modelDetails')]
+    scene:
+      Title: //div[contains(@class,'title')]/h2/text()
+      URL: //link[@rel="canonical"]/@href
+      Details: $details/div[@class='description']/p/text()
+      Image:
+        selector: //div[@class='player-thumb']/img/@src
+        postProcess:
+          - replace:
+              - regex: ^
+                with: https://tour.theflourishxxx.com
+      Date:
+        selector: $details//p[1]/text()
```

This needs to be a bit more exact
You get a lot of matches that the first happens to match the date in most of the scenes (set log to debug and do a scrape to get more info)
An example scene that fails is `https://tour.theflourishpov.com/trailers/Ryan-Rain-And-Lexie-Rose-Bath-Teaser.html`
```suggestion
        selector: $details//p[1]/text()[contains(.,"Added:")]
```